### ISSUE TITLE
ctrl+u (CutFromStart) seems broken

### ISSUE THREAD

**Issue by nibon7** [nushell/nushell#5470]
### Describe the bug

From commit 608b6f3634dc47ed37455414758147c65e7feb41, <kbd>ctrl</kbd>+<kbd>u</kbd> doesn't  behave like it used to.

<kbd>ctrl</kbd>+<kbd>u</kbd> now seems to be used to trigger user defined menus in `default_config.nu` 
```yml
    {
      name: commands_with_description
      modifier: control
      keycode: char_u
      mode: [emacs, vi_normal, vi_insert]
      event: { send: menu name: commands_with_description }
    }
```

### How to reproduce

1. start nu with default config
2. type something
3. press <kbd>ctrl</kbd>+<kbd>u</kbd>

### Expected behavior

<kbd>ctrl</kbd>+<kbd>u</kbd> can cut from the start of the buffer to the insertion point.

### Configuration

| key                | value                                    |
| ------------------ | ---------------------------------------- |
| version            | 0.62.0                                   |
| branch             | main                                     |
| tag                |                                          |
| short_commit       | 1cb449b2                                 |
| commit_hash        | 1cb449b2d14bcd77305024716db13a7dbecaede7 |
| commit_date        | 2022-05-07 12:33:33 +00:00               |
| build_os           | linux-x86_64                             |
| rust_version       | rustc 1.59.0 (gentoo)                    |
| cargo_version      | cargo 1.59.0                             |
| pkg_version        | 0.62.0                                   |
| build_time         | 2022-05-07 22:38:07 +08:00               |
| build_rust_channel | debug                                    |
| features           | default, trash, which, zip               |
| installed_plugins  |                                          |

**Comment by fdncred** [comment]
you can always reassign the menu keybinding from the default config. they're really there just for instructional purposes.

**Comment by nibon7** [comment]
Fixed by #5938. Close it.
